Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.yml

```diff
@@ -0,0 +1,60 @@
+category: Authentication
+commonfields:
+  id: ThycoticDSV
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.com)
+  name: url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Client ID for authenticate
```

Comment:
same here, just as a second example. but please review all descriptions and display names

Reply:
@guykeller 
It seems to have corrected all the descriptions.